docs/content/preview/yugabyte-cloud/cloud-secure-clusters/managed-ear.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Encryption at rest
+linkTitle: Encryption at rest
+description: YugabyteDB Managed cluster encryption at rest.
+headcontent: Encrypt your YugabyteDB cluster
+menu:
+  preview_yugabyte-cloud:
+    identifier: managed-ear
+    parent: cloud-secure-clusters
+    weight: 460
+type: docs
+---
+
+YugabyteDB Managed uses volume encryption for all data at rest, including your account data and your clusters and their backups. Data is AES-256 encrypted using native cloud provider technologies - S3 and EBS volume encryption for AWS, and server-side and persistent disk encryption for GCP. Volume encryption keys are managed by the cloud provider and anchored by hardware security appliances.
+
+In addition to cloud provider volume encryption, you can enable YugabyteDB encryption at rest (EAR) for clusters. When enabled, your cluster (including backups) is encrypted using a customer managed key (CMK) residing in a cloud provider Key Management Service (KMS). (Currently, only AWS KMS is supported.) You can grant YugabyteDB Managed access to the key with the requisite permissions to perform cryptographic operations using the key to secure the databases in your clusters.
```

Comment:
link to AWS KMS ? I know its helpful that you have linked it at the end, but it also helps to include it here.